### DISCUSSION TITLE
Save and Load models

### DISCUSSION THREAD

**Discussion by krinetic1234** [sebp/scikit-survival#179] [General]
Hi, how do you save and load models in sksurv. For ex, if I trained the gradient boosting model, but now i want to deploy in a flask app. i don't want to start training again. i just want to use it in my app.

**Comment by krinetic1234** [comment]
@sebp pls see above ^. Thanks!

**Comment by sebp** [comment]
The same way as for scikit-learn models, see 
https://scikit-learn.org/stable/modules/model_persistence.html
